src/algorithm/REDUCE_SUM-OMP.cpp

```diff
@@ -76,21 +76,48 @@ void REDUCE_SUM::runOpenMPVariant(VariantID vid, size_t RAJAPERF_UNUSED_ARG(tune
 
     case RAJA_OpenMP : {
 
-      startTimer();
-      for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
+      if (tune_idx == 0) {
+
+        startTimer();
+        for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
 
-        RAJA::ReduceSum<RAJA::omp_reduce, Real_type> sum(m_sum_init);
+          RAJA::ReduceSum<RAJA::omp_reduce, Real_type> sum(m_sum_init);
 
-        RAJA::forall<RAJA::omp_parallel_for_exec>(
-          RAJA::RangeSegment(ibegin, iend),
-          [=](Index_type i) {
-            REDUCE_SUM_BODY;
-        });
+          RAJA::forall<RAJA::omp_parallel_for_exec>(
+            RAJA::RangeSegment(ibegin, iend),
+            [=](Index_type i) {
+              REDUCE_SUM_BODY;
+          });
 
-        m_sum = sum.get();
+          m_sum = sum.get();
+
+        }
+        stopTimer();
 
       }
-      stopTimer();
+
+      if (tune_idx == 1) {
```

Comment:
Why not `else if`?

Reply:
Dunno. Just started with that and went with it.